sls-versions/src/main/java/com/palantir/sls/versions/SlsVersion.java

```diff
@@ -62,9 +62,43 @@ public static boolean check(String coordinate) {
 
     public abstract int getPatchVersionNumber();
 
-    public abstract OptionalInt firstSequenceVersionNumber();
+    /**
+     * The release candidate version number, if this version is a release candidate or release candidate snapshot.
+     */
+    public abstract OptionalInt rcNumber();
 
-    public abstract OptionalInt secondSequenceVersionNumber();
+    /**
+     * The snapshot version number, if this version is a release snapshot or release candidate snapshot.
+     */
+    public abstract OptionalInt snapshotNumber();
```

Comment:
This is a break too, for anybody extending `SlsVersion`, though as far as I can tell, this isn't done on any of our projects